pkg/bundle/create/create.go

```diff
@@ -254,6 +254,18 @@ func (o *Options) create(ctx context.Context, f createFunc) error {
 	return nil
 }
 
+// newBackendForConfig returns a Backend specified by config
+func (o *Options) newBackendForConfig(ctx context.Context, cfg v1alpha1.StorageConfig) (storage.Backend, error) {
+	dir := filepath.Join(o.Dir, config.SourceDir)
+	iface, err := storage.ByConfig(ctx, dir, cfg)
```

Comment:
Check this error here, since `iface` might be nil and return the cast error below, which hides the real error.

Reply:
Good catch! Added.